src/main/java/org/swisspush/redisques/util/RedisquesConfiguration.java

```diff
@@ -42,6 +42,7 @@ public class RedisquesConfiguration {
     private final int memoryUsageLimitPercent;
     private final int memoryUsageCheckIntervalSec;
     private static final int DEFAULT_CHECK_INTERVAL = 60; // 60s
+    private static final int DEFAULT_PROCESSOR_TIMEOUT = 240000; // 240S
```

Comment:
(*trivial*) Suggest:
```patch
-  ... DEFAULT_PROCESSOR_TIMEOUT ...
+  ... DEFAULT_PROCESSOR_TIMEOUT_MS ...
```
IMHO in such circumstances it is a good idea to make the unit more clear.